cmd/soroban-cli/src/config/identity/generate.rs

```diff
@@ -15,18 +15,35 @@ pub enum Error {
 pub struct Cmd {
     /// Name of identity
     pub name: String,
+
     /// Optional seed to use when generating seed phrase.
     /// Random otherwise.
     #[clap(long, short = 's')]
```

Comment:
just a thought - what about `p` for `phrase` instead of `s` so that it doesn't conflict with `S` (then `S` can be `s`)?

Reply:
This seed to generate the seed phrase and must be 16 characters long. Seed seems short enough in my opinion so perhaps just the other one gets 's'

Reply:
Fair enough, no strong opinion :)